ShellScript/Bash.sublime-syntax

```diff
@@ -448,31 +448,31 @@ contexts:
 
   cmd-declare-args:
     # option contains -f or -F
-    - match: ([-+])[aAgiIlnrtux]*[Ff][aAgiIlnrtux]*{{opt_break}}
+    - match: ([-+])[A-Za-z]*[Ff][A-Za-z]*{{opt_break}}
```

Comment:
At this point, is it worth just `([-+])(?i:[a-z]*f[a-z]*){{opt_break}}`? Or does the explicit upper/lower in every set help recognition?

I guess the pattern breaks down for the other ones that need explicit upper or lower case for the mode letter down below. Variable for `[A-Za-z]*?` then, maybe?

Reply:
General `(?i)` flag isn't that ideal, as `i` needs to be matched case-sensitive and all patterns should follow same style.

To apply same pattern as for ordinary options, we can however leverage `{{word_char}}` to replace `[A-Za-z]`.